src/audio/multiband_drc/user/multiband_drc.h

```diff
@@ -9,10 +9,11 @@
 #define __USER_MULTIBAND_DRC_H__
 
 #include <stdint.h>
-#include <user/crossover.h>
 #include <user/drc.h>
 #include <user/eq.h>
 
+#include "../../crossover/crossover_user.h"
```

Comment:
not sure how strongly crossover and multiband DRC are related, but maybe it would be better to extract required common parts and keep it in headers under src/include/sof/audio?

Reply:
for #include <user/drc.h>

it also need referenced by: ../../drc/drc_user.h, the structure is used from drc_user.h, combined with above, if put common to one header, seems also strange, image a header with crossover definition and drc structure definition, we may consult Seppo once he is back.